python-interface/src/MiniBotScripts/Note_Library.py

```diff
@@ -0,0 +1,122 @@
+import pygame.mixer as pm
+from pygame_play_tone import Note
+from time import sleep
+
+#Default volume for Notes
+DEFAULT_VOLUME=0.2
+
+# Notes that can be called on, where C4 is middle C
+C0 = 16.35
```

Comment:
I wrote the file, but I took the frequency values from a website. Should I cite that?